knative-operator/cmd/manager/main.go

```diff
@@ -66,13 +66,25 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Setup all Webhooks
```

Comment:
this is not longer exposed, to have a "custom" webhook, one needs to create it upfront and pass it to the manager. IMHO, might be good if somebody else can cross-check.